test/zepto.html

```diff
@@ -2664,11 +2670,16 @@
             })
           })
         })
-        t.assertTrue(fired)
-        t.assertIdentical(Zepto, arg1)
-        t.assertIdentical(Zepto, arg2)
-        t.assertIdentical(Zepto, arg3)
-        t.assertIdentical(Zepto, arg4)
+        t.pause()
+        setTimeout(function(){
+          t.resume(function(){
+            t.assertTrue(fired)
+            t.assertIdentical(Zepto, arg1)
+            t.assertIdentical(Zepto, arg2)
+            t.assertIdentical(Zepto, arg3)
+            t.assertIdentical(Zepto, arg4)
+          })
+        }, 500)
```

Comment:
This one is tricky. A timeout of `0` won't suffice because of the nested `ready()` invocations in the test. But, a timeout of `10` should suffice, making the test faster.